### PR TITLE
add batch

### PR DESCRIPTION
**Pull request by nguyenhonghai1997bg**

### CODE REVIEW

**Comment by nguyenhonghai1997bg** [comment]
https://edu-redmine.framgia.vn/issues/5300

File: app/Http/Controllers/Admin/BatchController.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+class BatchController extends Controller
+{
+    protected $batch;
+    
+    public function __construct(
+        BatchRepository $batch,
+        ProgramRepository $programRepository,
+        PositionRepository $positionRepository,
+        WorkspaceRepository $workspaceRepository
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/BatchController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/BatchController.php</br>**Line**: 23</br>The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line

File: app/Http/Controllers/Admin/BatchController.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    )
+    {
+        $this->batch = $batch;
+        $this->programRepository = $programRepository;
+        $this->positionRepository = $positionRepository;
+        $this->workspaceRepository = $workspaceRepository;
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/BatchController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/BatchController.php</br>**Line**: 29</br>Function closing brace must go on the next line following the body; found 1 blank lines before brace

File: app/Http/Controllers/Admin/BatchController.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+    public function update(BatchRequest $request, $id)
+    {
+        $this->batch->update($request->all());
+
+        return redirect()->route('batch.index');
+    }
+
+    public function show()
+    {
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/BatchController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/BatchController.php</br>**Line**: 58</br>Function closing brace must go on the next line following the body; found 1 blank lines before brace

File: app/Http/Controllers/Admin/BatchController.php [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+use App\Repositories\ProgramRepository;
+use App\Http\Requests\BatchRequest;
+
+class BatchController extends Controller
+{
+    protected $batch;
+    
+    public function __construct(
+        BatchRepository $batch,
+        ProgramRepository $programRepository,
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/BatchController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.CloseBracketLine</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/BatchController.php</br>**Line**: 21</br>The closing parenthesis of a multi-line function declaration must be on a new line

File: app/Http/Controllers/Admin/BatchController.php [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        PositionRepository $positionRepository,
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/BatchController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/BatchController.php</br>**Line**: 22</br>The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line

**Pull request merged**